README.md

```diff
@@ -138,8 +138,9 @@ The simplest way is to [![install with bioconda](https://img.shields.io/badge/in
 It can also be installed with `brew` as `brew install brewsci/bio/mosdepth` or used via docker with quay:
 ```
 docker pull quay.io/biocontainers/mosdepth:0.2.4--he527e40_0
-docker run -v /hostpath/:/opt/mount quay.io/biocontainers/mosdepth:0.2.4--he527e40_0 mosdepth -n --fast-mode -t 4 --by 1000 /opt/mount/sample $bam
+docker run -v /hostpath/:/opt/mount quay.io/biocontainers/mosdepth:0.2.4--he527e40_0 mosdepth -n --fast-mode -t 4 --by 1000 /opt/mount/sample /opt/mount/$bam
 ```
+`/hostpath/` is the path to the directory containing input file 
```

Comment:
can you remove this line? I don't want this section getting any longer and this will be apparent to anyone who uses docker.

Reply:
Done